src/components/Grid/SimpleGrid.stories.tsx

```diff
@@ -0,0 +1,241 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+
+import SimpleGrid from "./SimpleGrid";
+import Card, { CardHeading, CardContent } from "../Card/Card";
+import Icon from "../Icons/Icon";
+import { iconNamesArray } from "../Icons/iconVariables";
+import Image from "../Image/Image";
+
+const meta: Meta<typeof SimpleGrid> = {
+  title: "Components/Page Layout/SimpleGrid",
+  component: SimpleGrid,
+  decorators: [withDesign],
+  argTypes: {
+    className: { control: false },
+    columns: {
+      control: { min: 2, type: "number" },
+    },
+    gap: {
+      control: { type: "select" },
+      table: { defaultValue: { summary: "grid.l" } },
+      options: [
+        "grid.xxs",
```

Comment:
Update the definition for `GridGaps` to use an `as const` array that can be reused here.

Reply:
Thanks.  As I was working on `Form`, I realized I had forgotten this.